musicdiff/visualization.py

```diff
@@ -668,6 +668,12 @@ def show_diffs(score1: m21.stream.Score, score2: m21.stream.Score):
         else:
             score2.metadata.composer = "score2          " + originalComposer2
 
-        score1.show("musicxml.pdf", makeNotation=False)
-        score2.show("musicxml.pdf", makeNotation=False)
+        #save files if requested
+        if (out_path1 is not None) and (out_path2 is not None):
+            score1.write("musicxml.pdf", fp=out_path1)
+            score2.write("musicxml.pdf", fp=out_path2)
```

Comment:
makeNotation=False
(otherwise music21 will run through the score, changing all sorts of stuff before doing the export)